src/EditorFeatures/TestUtilities/AbstractCommandHandlerTestState.cs

```diff
@@ -28,105 +31,79 @@ internal abstract class AbstractCommandHandlerTestState : IDisposable
         public readonly IEditorOperations EditorOperations;
         public readonly ITextUndoHistoryRegistry UndoHistoryRegistry;
         private readonly ITextView _textView;
+        private readonly DisposableTextView? _createdTextView;
         private readonly ITextBuffer _subjectBuffer;
 
         public AbstractCommandHandlerTestState(
             XElement workspaceElement,
-            IList<Type> excludedTypes = null,
-            ComposableCatalog extraParts = null,
-            string workspaceKind = null)
-            : this(workspaceElement, GetExportProvider(excludedTypes, extraParts), workspaceKind)
+            ComposableCatalog extraParts,
+            string? workspaceKind = null)
+            : this(workspaceElement, GetExportProvider(excludedTypes: null, extraParts), workspaceKind)
         {
         }
 
-        /// <summary>
-        /// This can use input files with an (optionally) annotated span 'Selection' and a cursor position ($$),
-        /// and use it to create a selected span in the TextView.
-        /// 
-        /// For instance, the following will create a TextView that has a multiline selection with the cursor at the end.
-        /// 
-        /// Sub Goo
-        ///     {|Selection:SomeMethodCall()
-        ///     AnotherMethodCall()$$|}
-        /// End Sub
-        ///
-        /// You can use multiple selection spans to create box selections.
-        ///
-        /// Sub Goo
-        ///     {|Selection:$$box|}11111
-        ///     {|Selection:sel|}111
-        ///     {|Selection:ect|}1
-        ///     {|Selection:ion|}1111111
-        /// End Sub
-        /// </summary>
         public AbstractCommandHandlerTestState(
```

Comment:
Why did we remove the box selection documentation? It still works right?

Reply:
Oh right, d'oh. Good catch.